docs/modelserving/storage/gcs/gcs.md

```diff
@@ -17,36 +17,56 @@ e.g. ```gs://kfserving-examples/models/tensorflow/flowers```
 
 KServe supports authenticating using Google Service Account Key
 
-### Create a Service Account Key
+### Create a Google Service Account Key
 
 * To create a Service Account Key follow the steps [here](https://cloud.google.com/iam/docs/keys-create-delete#iam-service-account-keys-create-console).
 * Base64 encode the generated Service Account Key file
 
 
-## Create Google Secret
+## Create Secret and attach to Service Account
+
 
 ### Create secret
+
+You can create a secret using either the imperative kubectl command or by declaratively defining it in a YAML file.
+
+#### Using Imperative Command
+
+=== "kubectl"
+```bash
+kubectl create secret generic gcscreds --from-file=gcloud-application-credentials.json=/path/to/gcloud-application-credentials.json
+```
+
+#### Using Declarative YAML
+
 === "yaml"
 ```yaml
 apiVersion: v1
 kind: Secret
 metadata:
-  name: storage-config
+  name: gcscreds
 type: Opaque
-stringData:
-  gcs: |
-    {
-      "type": "gs",
-      "bucket": "mlpipeline",
-      "base64_service_account": "c2VydmljZWFjY291bnQ=" # base64 encoded value of the credential file
-    }
+data:
+  gcloud-application-credentials.json: <base64 encoded value of the credential file>
```

Comment:
It would be nice to have a note that this name can be configured using the inferenceservice-config configmap. https://github.com/kserve/kserve/blob/3deb30fee58d63e772db28abf58582570dbb28cc/config/configmap/inferenceservice.yaml#L113